spinners.json

```diff
@@ -1278,5 +1278,25 @@
 			"βββββρβ",
 			"ββββββρ"
 		]
+	},
+		"fish": {
+		"interval": 150,
+		"frames": [
+			">))'>",
+			"<'((<"
+		]
+	},
+			"futurebar": {
```

Comment:
```suggestion
	"futurebar": {
```

Reply:
I think it would be better to use a descriptive name.

Reply:
like, aesthetic ?